src/CRTM_Forward_Module.f90

```diff
@@ -602,15 +602,15 @@ FUNCTION profile_solution (m, Opt, AncillaryInput) RESULT( Error_Status )
       IF ( CRTM_CloudCoeff_IsLoaded() .AND. &
            (RTV(1)%n_Stokes > 1 .AND. CloudC%N_PHASE_ELEMENTS < 6 )) THEN
          Error_Status = FAILURE
-         WRITE( Message,'("N_PHASE_ELEMENTS OF CLOUD LUT NOT RIGHT ",i0)' ) CloudC%N_PHASE_ELEMENTS
+         WRITE( Message,'("N_PHASE_ELEMENTS OF CLOUD LUT NOT RIGHT ",i0)' ) CloudC%N_PHASE_ELEMENTS, AeroC%N_PHASE_ELEMENTS, RTV(1)%n_Stokes
          CALL Display_Message( ROUTINE_NAME, Message, Error_Status )
          RETURN
       END IF
 
       IF ( CRTM_AerosolCoeff_IsLoaded() .AND. &
            (RTV(1)%n_Stokes > 1 .AND. AeroC%N_PHASE_ELEMENTS < 6 )) THEN
          Error_Status = FAILURE
-         WRITE( Message,'("N_PHASE_ELEMENTS OF AEROSOL LUT NOT RIGHT ",i0)' ) AeroC%N_PHASE_ELEMENTS
+         WRITE( Message,'("N_PHASE_ELEMENTS OF AEROSOL LUT NOT RIGHT ",i0)' ) AeroC%N_PHASE_ELEMENTS, AeroC%N_PHASE_ELEMENTS, RTV(1)%n_Stokes
```

Comment:
Same here. It makes sense to add `RTV(1)%n_Stokes`, but there is no need to print `CloudC%N_PHASE_ELEMENTS` here. (AeroC%N_PHASE_ELEMENTS is added twice, likely a typo)

The IF statement at line 618 makes sure `CloudC%N_PHASE_ELEMENTS == AeroC%N_PHASE_ELEMENTS` if they are both loaded.